src/nvim/tui/tui.c

```diff
@@ -1429,6 +1431,18 @@ void tui_option_set(TUIData *tui, String name, Object value)
   }
 }
 
+void tui_term_unhandled(TUIData *tui, String cmd)
```

Comment:
doesn't look like this is used anywhere?

Reply:
Do you think this PR is still relevant considering the changes in the past months? (specially the changes in libvterm)
Also we have [this PR](https://github.com/neovim/neovim/pull/22159) which I think is better approaching this issue.
If this PR is still relevant I'm willing to rebase it and continue working on it.

Reply:
Yes. Going to try to merge https://github.com/neovim/neovim/pull/22159 today